cmd/xzcatomicswap/main.go

```diff
@@ -404,6 +404,58 @@ func createSig(tx *wire.MsgTx, idx int, pkScript []byte, addr xzcutil.Address,
 	return sig, wif.PrivKey.PubKey().SerializeCompressed(), nil
 }
 
+func dpk(c *rpc.Client, addr xzcutil.Address) (wif *xzcutil.WIF, err error) {
+	addrStr := addr.EncodeAddress()
+	if !addr.IsForNet(chainParams) {
+		return nil, fmt.Errorf("address %v is not intended for use on %v",
+			addrStr, chainParams.Name)
+	}
+	param0, err := json.Marshal(addrStr)
+	if err != nil {
+		return nil, err
+	}
+	params := []json.RawMessage{param0}
+	// This should always fail the first time as Zcoin added a one-time authoriz-
+	// ation key returned in error string. Along with a warning. The idea is that
+	// inexperienced people are warned if scammers propose they use `dumpprivkey'
+	_, err = c.RawRequest("dumpprivkey", params)
+	if err == nil {
+		unexpected := errors.New("dpk: No authorization challenge")
+		return nil, unexpected
+	}
+
+	errStr := err.Error()
+	searchStr := "authorization code is: "
+	i0 := strings.Index(errStr, searchStr)
+	if i0 == -1 {
+		return nil, err
+	}
+	i := i0 + len(searchStr)
+	authStr := errStr[i : i+4]
+	//
```

Comment:
Either remove this or add some text to the comment

Reply:
Please clarify:
You mean remove the text inside the function"
```
	// This should always fail the first time as Zcoin added a one-time authoriz-
	// ation key returned in error string. Along with a warning. The idea is that
	// inexperienced people are warned if scammers propose they use `dumpprivkey'
```
...and add above the function where go usually asks for comments?

Reply:
This stray comment has no text.  If it's unnecessary, remove it.  Otherwise, if you wanted to comment about something specifically on this line, do so.

Reply:
Oh, ok I understand .. it was just a stylistic thing I do. Sure I will re-submit with this line removed.

Reply:
New PR: https://github.com/decred/atomicswap/pull/107 replaces this one which I will close.

Reply:
please update PRs by pushing new commits instead of opening new PRs to make review changes

Reply:
I always fork your repo only to fix stuff then delete it so that would not be possible

Reply:
Why not possible? Even if you nuke your fork, you can fetch this PR from the repo it was opened to and reinstate the branch.